tests/unit/FolioNumber.spec.ts

```diff
@@ -1,45 +1,98 @@
 // Libraries
 import Vue from 'vue'
 import Vuetify from 'vuetify'
-
-// Store
-import { getVuexStore } from '@/store'
-
-// Components
+import { mount } from '@vue/test-utils'
+import flushPromises from 'flush-promises'
 import { FolioNumber } from '@/components/DefineCompany'
-import { shallowMount } from '@vue/test-utils'
 
 Vue.use(Vuetify)
-
 const vuetify = new Vuetify({})
-const store = getVuexStore()
 
 describe('Folio number component', () => {
   it('displays editing view', () => {
-    store.state.stateModel.nameRequest.entityType = 'BC'
-    store.state.stateModel.accountInformation.accountType = 'PREMIUM'
-    const wrapper = shallowMount(FolioNumber, {
```

Comment:
Deep mount is needed so that v-text-field is fully rendered (instead of just stubbed).